### DISCUSSION TITLE
Cmd + Click on function or class takes you there

### DISCUSSION THREAD

**Discussion by amirgamil** [RecordReplay/devtools#4017] [Ideas]
Same gist as VSCode and most editors - Cmd + Click on a function/class should take you there. I keep switching between Replay to debug and VSCode to track where I should debug. Ideally, it should all be in replay.

**Comment by jasonLaster** [comment]
Thanks! That's one of our goals as well

**Comment by jonbell-lot23** [comment]
March 15, 2022:

We've talked about this (and items like it) a lot. Is it worth promoting into a task so we can get this done?
